packages/vue3-vite/src/generators/library/files/src/index.ts.template

```diff
@@ -1,3 +1,3 @@
 export {
-  Widget
-} from '@lib/Widget.vue'
\ No newline at end of file
+  MyWidget
+} from '@lib/MyWidget.vue'
```

Comment:
I have to give it a multi-word name to make the lint test pass.
`Cookies` and `Home` are also renamed to `CookiesConsent` and `HomePage` respectively for the same reason.